superset-frontend/src/components/AlteredSliceTag.jsx

```diff
@@ -120,16 +152,10 @@ export default class AlteredSliceTag extends React.Component {
         })
         .join(', ');
     }
-    if (
-      this.state.controlsMap[key] &&
-      this.state.controlsMap[key].type === 'BoundsControl'
-    ) {
+    if (controlsMap[key] && controlsMap[key].type === 'BoundsControl') {
       return `Min: ${value[0]}, Max: ${value[1]}`;
     }
-    if (
-      this.state.controlsMap[key] &&
-      this.state.controlsMap[key].type === 'CollectionControl'
-    ) {
+    if (controlsMap[key] && controlsMap[key].type === 'CollectionControl') {
```

Comment:
you can use optional chaining here and line 155 as well.

Reply:
Done